data/helpers.d/backend

```diff
@@ -442,7 +442,7 @@ EOF
   ynh_store_file_checksum "$finalfail2banjailconf"
   ynh_store_file_checksum "$finalfail2banfilterconf"
 
-  systemctl try-reload-or-restart fail2ban
+  ynh_systemd_action --service_name=fail2ban --action=reload
```

Comment:
Well,

I think `reload` is not same than `try-reload-or-restart`.

And for this helper maybe we need to manage the error because actually some app install/upgrade fail juste because the fail2ban fail to reload (see https://github.com/YunoHost-Apps/synapse_ynh/issues/88#issuecomment-478808204)

Reply:
https://github.com/YunoHost/yunohost/blob/stretch-unstable/data/helpers.d/system#L116-L119
https://www.freedesktop.org/software/systemd/man/systemctl.html#reload-or-restart%20PATTERN%E2%80%A6